logic/tool/TypeResolver.java

```diff
@@ -269,15 +269,9 @@ private TypeVariable register(ThingVariable var) {
             var.has().forEach(constraint -> registerHas(resolver, constraint));
             if (insertable) var.relation().forEach(constraint -> registerInsertableRelation(resolver, constraint));
             else var.relation().forEach(constraint -> registerRelation(resolver, constraint));
-            var.iid().ifPresent(constraint -> registerIID(resolver, constraint));
             return resolver;
         }
 
-        private void registerIID(TypeVariable resolver, IIDConstraint iidConstraint) {
-            Thing thing = conceptMgr.getThing(iidConstraint.iid());
-            if (thing != null) traversal.labels(resolver.id(), thing.getType().getLabel());
-        }
```

Comment:
solving IIDs into types has no benefit, as its already as specialised as it can be